0.3.0/android/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Default image that can build OpenRCT2 for Android.
```

Comment:
This whole file is suspicious to me, there should be no need to install host libraries to build for android. This will need some inspection and updates in future PR.

Reply:
I don't know what you mean by host libraries.